### PR TITLE
[IMP] Add SASS & Compass install sample

### PR DESCRIPTION
**Pull request by lasley**
https://github.com/OCA/social/pull/179#issuecomment-314146715

### CODE REVIEW

File: sample_files/.travis.yml [hunk 1]
```diff
@@ -30,6 +30,12 @@ addons:
 #  - "export DISPLAY=:99.0"
 #  - "sh -e /etc/init.d/xvfb start"
 
+# Sometimes complicated web repos need Compass & SaSS:
```
Comment (pedrobaeza) [inline sample_files/.travis.yml]: s/web/website

**Comment by lasley** [comment]
Good catch @pedrobaeza - done. I went ahead and squashed into original seeing as it was simple.

**Comment by moylop260** [comment]
Extra info:
We have working this feature from: https://github.com/OCA/website/pull/284
Thanks to t2d contribution https://github.com/Vauxoo/travis2docker/pull/77 by @lasley 🌮

**Comment by lasley** [comment]
I knew I remembered fixing this permanently somewhere! This also explains why the difference between Runbot and Travis results!

**Pull request merged**